dev/Scroller/InteractionTests/ScrollerTestsWithInputHelper.cs

```diff
@@ -736,6 +736,156 @@ public void StretchImageWithMouseWheel()
+                Log.Comment("scroller51.VerticalScrollPercent={0}", scroller51.VerticalScrollPercent);
+
+                // No layout offset is expected
+                double contentLayoutOffsetX = 0.0;
+                double contentLayoutOffsetY = 0.0;
+
+                GetScrollerContentLayoutOffset(out contentLayoutOffsetX, out contentLayoutOffsetY);
+
+                if (!success ||
+                    (isForHorizontalDirection && (scroller51.HorizontalScrollPercent == 0.0 || scroller51.HorizontalScrollPercent == 100.0)) ||
+                    (isForHorizontalDirection && scroller51.VerticalScrollPercent != 0.0) ||
+                    (!isForHorizontalDirection && scroller51.HorizontalScrollPercent != 0.0) ||
+                    (!isForHorizontalDirection && (scroller51.VerticalScrollPercent == 0.0 || scroller51.VerticalScrollPercent == 100.0)) ||
+                    contentLayoutOffsetX != 0.0 ||
+                    contentLayoutOffsetY != 0.0)
+                {
+                    LogAndClearTraces(recordWarning: false);
+                }
+
+                Log.Comment("Overpan Scroller to trigger content shrinkage");
+                PrepareForScrollerManipulationStart("scroller51");
+
+                startPoint = isForHorizontalDirection ?
+                        new Point(scroller51UIObject.BoundingRectangle.Left + 240, scroller51UIObject.BoundingRectangle.Top + 100) :
+                        new Point(scroller51UIObject.BoundingRectangle.Left + 100, scroller51UIObject.BoundingRectangle.Top + 240);
+                endPoint = isForHorizontalDirection ?
+                        new Point(scroller51UIObject.BoundingRectangle.Left + 10, scroller51UIObject.BoundingRectangle.Top + 100) :
+                        new Point(scroller51UIObject.BoundingRectangle.Left + 100, scroller51UIObject.BoundingRectangle.Top + 10);
+
+                InputHelper.Pan(obj: scroller51UIObject, start: startPoint, end: endPoint);
+
+                Log.Comment("Waiting for scroller51 pan completion");
+                success = WaitForManipulationEnd("scroller51", "txtScrollerState");
+
+                Log.Comment("scroller51.HorizontalScrollPercent={0}", scroller51.HorizontalScrollPercent);
+                Log.Comment("scroller51.VerticalScrollPercent={0}", scroller51.VerticalScrollPercent);
+
+                // Layout offset is expected
+                GetScrollerContentLayoutOffset(out contentLayoutOffsetX, out contentLayoutOffsetY);
+
+                if (!success ||
+                    (isForHorizontalDirection && scroller51.HorizontalScrollPercent != 100.0) ||
+                    (isForHorizontalDirection && scroller51.VerticalScrollPercent != 0.0) ||
+                    (!isForHorizontalDirection && scroller51.HorizontalScrollPercent != 0.0) ||
+                    (!isForHorizontalDirection && scroller51.VerticalScrollPercent != 100.0) ||
+                    (isForHorizontalDirection && contentLayoutOffsetX != 30.0) ||
```

Comment:
I don't really know what the logic is behind all these numbers. 30, 40, 10, 100, 240... they look like they're combinations of something but it's not obvious what.